test/lish/package_creation_provides_invalid_json_test.dart

```diff
@@ -10,12 +10,13 @@ import '../test_pub.dart';
 import 'utils.dart';
 
 void main() {
-  setUp(d.validPackage.create);
-
   test('package creation provides invalid JSON', () async {
     await servePackages();
+    await d.validPackage.create();
     await d.credentialsFile(globalServer, 'access token').create();
-    var pub = await startPublish(globalServer);
+    var pub = await startPublish(
+      globalServer,
+    );
```

Comment:
Do we need this change? Don't get me wrong I love trailing comma :D
```suggestion
    var pub = await startPublish(globalServer);
```

Reply:
No we don't - for a while I thought I had to add arguments here (because I needed a custom _PUB_TEST_SDK_VERSION)